torch/csrc/api/include/torch/nn/options/loss.h

```diff
@@ -90,7 +92,7 @@ struct TORCH_API TripletMarginLossOptions {
   /// E. Riba et al. Default: False
   TORCH_ARG(bool, swap) = false;
   /// Specifies the reduction to apply to the output. Default: Mean
-  TORCH_ARG(Reduction::Reduction, reduction) = Reduction::Mean;
+  TORCH_ARG(10::variant<enumtype::kNone, enumtype::kMean, enumtype::kSum>, reduction) = torch::kMean;
```

Comment:
Sorry I was looking if a PR of mine was in the list and found this PR which kind of concerns what I was doing :P I think you are missing a `c` here (should be `c10`) and in previous declarations too!

Reply:
@CarMiranda Thanks a lot for the catch! I just fixed it. After this PR is merged, I will do a sweep to change all `torch::nn` layers that use `Reduction` to use the corresponding variant type. :D